quickfix-app-messages.xml

```diff
@@ -0,0 +1,10 @@
+<capture-points>
+  <capture-point enabled="true" 
+  
+    class-name="quickfix.Session" method-name="toApp" 
+    insert-class-name="quickfix.Session" insert-method-name="next" 
+
+    capture-key-expression="message.getHeader().getField(49).getValue() + &quot;-&gt;&quot; + message.getHeader().getField(56).getValue() + &quot;#&quot; + message.getHeader().getField(34).getValue()" 
```

Comment:
consider using fields instead of method calls, it is much faster.
Debugger can access even private fields.